asset_stream_manager/session.h

```diff
@@ -32,17 +32,29 @@ class CdcFuseManager;
 class ProcessFactory;
 class Process;
 
-// Manages the connection of a workstation to a single gamelet.
+// Defines a remote target and how to connect to it.
+struct SessionTarget {
+  // SSH username and hostname of the remote target, formed as [user@]host.
+  std::string user_host;
```

Comment:
This struct makes combining user and host into one field even look weirder.

Reply:
I can split them in another CL. Shouldn't be too much work.